scripts/repo.js

```diff
@@ -182,11 +118,13 @@ function buildDocYML (sitemap, baseDir='', levelCount=0) {
 async function buildDocs (pathToDocs) {
   const mkdocsTemplate = Handlebars.compile(await fs.readFile(path.resolve(__dirname, '..', 'mkdocs.yml'),  'utf8'));
   const themeDir = path.resolve(__dirname, '..', 'cinder');
+  const sitemapJSON =  await fs.readFile(path.resolve(pathToDocs, 'toc.json'), 'utf8');
```

Comment:
I thought this returns a Buffer rather than string

Reply:
Adding 'utf8' as the second argument turns it into a string.

Reply:
you could just `require` it. then it doesn't matter whether it's js or json, and it turns it into an obj